src/PlanView/PlanView.qml

```diff
@@ -53,6 +53,7 @@ Item {
     property int    _toolStripBottom:                   toolStrip.height + toolStrip.y
     property var    _appSettings:                       QGroundControl.settingsManager.appSettings
     property var    _planViewSettings:                  QGroundControl.settingsManager.planViewSettings
+    property bool   _enableROI:                         _appSettings ? _appSettings.enableROI : false
```

Comment:
You shouldn't need the null check here. The settings objects area always available